cardano_node_tests/tests/tests_conway/test_conway_extras.py

```diff
@@ -0,0 +1,925 @@
+        * Submit certificate
+        * check it is not possible to retire before register
+        """
+        temp_template = common.get_test_id(cluster)
+        drep_keys = cluster.g_conway_governance.drep.gen_key_pair(
+            key_name=temp_template, destination_dir="."
+        )
+        deposit = cluster.conway_genesis["dRepDeposit"]
+
+        reqc.cli089.start(url=helpers.get_vcs_link())
+        ret_cert = cluster.g_conway_governance.drep.gen_retirement_cert(
+            cert_name=temp_template,
+            deposit_amt=deposit,
+            drep_vkey_file=drep_keys.vkey_file,
+        )
+        tx_files_ret = clusterlib.TxFiles(
+            certificate_files=[ret_cert],
+            signing_key_files=[payment_addr.skey_file, drep_keys.skey_file],
+        )
+
+        # For handling case where str method of submit api error returns extra \\
+        error_string_content = ""
+
+        # Expecting error for both cases as drep is not registered
+        if submit_method == "api":
+            with pytest.raises(submit_api.SubmitApiError) as excinfo_submit:
+                clusterlib_utils.build_and_submit_tx(
+                    cluster_obj=cluster,
+                    name_template=f"{temp_template}_ret",
+                    src_address=payment_addr.address,
+                    submit_method=submit_method,
+                    use_build_cmd=use_build_cmd,
+                    tx_files=tx_files_ret,
+                    deposit=-deposit,
+                )
+            error_string_content = "\\"
+        else:
+            with pytest.raises(clusterlib.CLIError) as excinfo_cli:
+                clusterlib_utils.build_and_submit_tx(
+                    cluster_obj=cluster,
+                    name_template=f"{temp_template}_ret",
+                    src_address=payment_addr.address,
+                    submit_method=submit_method,
+                    use_build_cmd=use_build_cmd,
+                    tx_files=tx_files_ret,
+                    deposit=-deposit,
+                )
+
+        drep_id = cluster.g_conway_governance.drep.get_id(
+            drep_vkey_file=drep_keys.vkey_file,
+            out_format="hex",
+        )
+        err_msg = str(excinfo_submit.value if excinfo_submit else excinfo_cli.value)
+        assert (
+            f'ConwayDRepNotRegistered (KeyHashObj (KeyHash {error_string_content}"{drep_id}'
+            in err_msg
+        ), err_msg
+        reqc.cli089.success()
+
+    @allure.link(helpers.get_vcs_link())
+    @submit_utils.PARAM_SUBMIT_METHOD
+    @common.PARAM_USE_BUILD_CMD
+    @pytest.mark.testnets
+    @pytest.mark.smoke
+    def test_drep_no_multiple_registration(
+        self,
+        cluster: clusterlib.ClusterLib,
+        payment_addr: clusterlib.AddressRecord,
+        use_build_cmd: bool,
+        submit_method: str,
+    ):
+        """Test Drep cannot be registered multiple time.
+
+        * Create a drep registration certificate
+        * Submit certificate
```

Comment:
Update description e.g. Submit certificate multiple times

Reply:
Thank you for the feedback. The description has been updated and moved to test_drep.py.